src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,132 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+    /** The tasks in this task list. */
+    private List<Task> tasks;
+
+    /**
+     * Creates an empty task list with no tasks.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Creates a task list with the given tasks.
+     * @param tasks The tasks to be added.
+     */
```

Comment:
This comment is missing the empty line between description and parameter section as stated in the coding convention page. Happens a few times in this class, so I'll just comment once here!

Reply:
Noted!